 `K202401251552` methylation staring point
 
 Key words: `Depmap` `methylation`  `MethylationToActivity` `plan`
 
 Related Zettle ID: [K202305191223_methy_NMF](https://github.com/yz46606/zettle_yz/blob/main/K202305191223_methy_NMF.md) 
 
# Question

How to get a reliable TE annotation results? search and decide a good TE annotation plan and test if it works.

How to make a reliable intron alignment results and show that? may be need to provide the alignment similarity score, and compared that with the coding sequence? the alignment itself is hard to show, because too long, to many gaps.
also for species have a totally different gene structure and intron distribution, how to select the introns that used for alignment?


# related Email content

Me:
one thing I think not very applicable is to do the alignment of the 7th intron sequence for all 34 species. it is possible for Bra family species, which I have already done that, may be I can make the analysis and statistic more accuracy, but as you see the distribution of JmJc domain for all species in Figure 3, for species outside Bra, 
the methylated intron may not located in the 7th intron, it is hard to tell which intron I need to selected for alignment with that in Arabidopsis, for example.

Bob & Hosung:

he worries about how long the repeat can covers the intron.

For example, just imagine that the repeat is only 20-30 bp. This is too small repeat and only part of the long intron sequence.

We can’t say that this repeat represents the whole characteristic of the intron of that species.

This maybe just an artifact of the repeat program you used.

I think he wants to see the annotation of TEs/repeats in the intron sequence of each species.

Eg) B. rapa has TE1, TE2, and SINE, and the length of each TEs are 300bp, 400bp, 20bp.

 

So, Bob wants to see alignment of the 7th intron sequence to show they are not conserved.

And then we need to confirm that the intron sequence is totally different between species.

You already showed that they have different types of repeats, but if those repeats are just small fragments and they are only very small portion of the methylated intron, maybe that is an artifact.

 

maybe we can have a zoom with Bob, if it doesn’t make sense.

 

Adding to #2, is there any other public data we could fold into our analysis, what paper might we have missed etc? the paper use WGBS, so what about different microarray data? 450, 850, 950 etc.


# Results summary  

# Related papers that may help
the paper do the benchmark of current TE annotation software and come up with a pipeline to get intact TE annotation library, so may be what I need.

Related paper note ID: [N202401251601_MethylationtoActivity](https://github.com/yz46606/paper_note/blob/main/N202401251601_MethylationtoActivity.md)
