 `K202304281321` Enrichment of SLAM-seq DEGs in SMAD4/9 RARA binding genes
 
 Key words: `DEG differential expression gene` `ChIP-seq` `SLAM-seq` `enrichment` `SMAD9/4` `RARA` 
 
# Question

If there is a significant overlap between the differentially expressed genes (DEGs) identified from 1-hour RA treatment cell lines by SLAM-seq and SMAD4/9 RARA binding genes? This idea is similar to the analysis of the overlap between DEGs identified from RNA-seq and SMADs/RARA ChIP-seq. Related Zettle ID: [K202304281321_OverlapEnrich.md](https://github.com/yz46606/zettle_yz/blob/main/K202304281321_OverlapEnrich.md)  

It may be challenging to perform this analysis due to potential issues with the quality of the SLAM-seq data. Therefore, it may not yield reliable results.

To investigate whether SMADs/RARA regulate genes at an early time point of 1-hour RA exposure, it would be beneficial to explore enrichment between SLAM and ChIP-seq in this analysis.

Additionally, it would be helpful to perform function enrichment analysis on the identified SLAM DEGs to investigate major pathway changes at the early time point. This would provide another valuable insight into how these genes are regulated and contribute to the cellular response to RA exposure.

# Results summary  

Related Work log ID: [W202304281527_SLAMEnrich.md](https://github.com/yz46606/Working_record/blob/main/W202304281527_SLAMEnrich.md)

DEG genes is not significantly overlaped with peaks, since overall genes (~1800 genes) with accumulated expression in SLAM-seq already show a significant enrichment with RARA binding peaks. so use those 1800 genes as background ratio will cover up the signal of DEG genes.  My hypothesis to explain this observation is that, if those 1800 genes can stably and quickly gain new transcripts in a short time across all 5 batched, it means that they are transcription active genes in this cell type, so probably also more likely to be bound with TFs.
