 `K202304281214` IBM1 expression variation at species level 
 
 Key words: `IBM1 othologous` `34 species` `angiosperm` `RNA-seq` `isoform quantification` `methylation demethylation enzyme` 
 
# Question

Intron methylation variation in IBM1 orthologous seems frequently happened across angiosperm species. it is not known that if the intron
methylation in other species can affect IBM1 transcription process as that happened in Arabidopsis. so it would be very worthy to explore 
`short/long isoform expression levels of other species, especielly those within the Eutrema cluster`.

Problem is how to defined short/long isoform in other species? that would be an technique issue to think about. I can start to try from the T.parvula. it turned out that T.parvular although have a long intron and almost lost all methylation comparing other species in Bra, this seems does not affect its transcription process. this observation does not fit with my expectation, since based on my hypothesis, if things consistent with that in Arabidopsis, T.parvular should have a high proportion of short isoform. however, when further think about it, the observation make sense. methylation controled regulated can hardly be a stable mechanism, since DNA methylation is not stable.

it seems that the regulation mechnism only hold in arabidopsis....


so that to summarize, the things I can do is:
1. check the behavious of isoform ratio for the 9 species within bra, propably it is enough for this gene expression analysis.
2. figure out a way to represent the data.


# Results summary  

Related Work log ID: [W202304281230_SpeciesEXP.md](https://github.com/yz46606/Working_record/blob/main/W202304281230_SpeciesEXP.md)

