 `K202401251552` methylation staring point
 
 Key words: `Depmap` `methylation`  `MethylationToActivity` `plan`
 
 Related Zettle ID: [K202305191223_methy_NMF](https://github.com/yz46606/zettle_yz/blob/main/K202305191223_methy_NMF.md) 
 
# Question

1: Work with Charlie to figure out what we already have, where is the data etc.

 

2: look into getting Xiang’s pipeline run on our data: https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02220-y

 

3: look into figuring out what methylation data was already generated at the Sanger Institute, e.g. how many pediatric cell lines, how many neuroblastoma, how many with paired drug data, how many with paired CRISPR data, can the data be easily accessed etc: https://www.cancerrxgene.org/downloads/bulk_download

PeDep currently do not have methylation data and comet have 300 methylation data, but have no other matched drug or crispr data, so not very useful.  
 

Adding to #2, is there any other public data we could fold into our analysis, what paper might we have missed etc? the paper use WGBS, so what about different microarray data? 450, 850, 950 etc.


4: 

 for some of these cell lines where it is ambiguous what it is, there will likely be much more information available on Cellosaurus: 
https://www.cellosaurus.org/

CRISPR data seems to be here. The Sanger’s CRSPR data is also probably (mostly) available on DepMap.org (but its hard to know exactly what has been moved where….).
https://cellmodelpassports.sanger.ac.uk/downloads
 

Drug = Temozolomide. Gene = MGMT. MGMT “promotor methylation” should predict Temozolomide sensitivity (IC50).

 

CRISPR data from Broad Institute (DepMap) is here: DepMap Data Downloads
https://depmap.org/portal/download/all/


Info/paper From Anang Shelat about MGMT:

 

Here are two papers describing the use of methylation data to predict MGMT status and response to TMZ.

 

MGMT methylation analysis of glioblastoma on the Infinium methylation BeadChip identifies two distinct CpG regions associated with gene silencing and outcome, yielding a prediction model for comparisons across datasets, tumor grades, and CIMP-status

https://link.springer.com/article/10.1007/s00401-012-1016-2?utm_source=getftr&utm_medium=getftr&utm_campaign=getftr_pilot

 

Sensitivity Analysis of the MGMT-STP27 Model and Impact of Genetic and Epigenetic Context to Predict the MGMT Methylation Status in Gliomas and Other Tumors

https://www.sciencedirect.com/science/article/pii/S1525157816000477?via%3Dihub#bib11

 

The actual model is a simple logistic regression using 2 probe sites M-scores:

Logit(y) = 4:3215 + 0.5271 * cg12434587 + 0:9265 * cg12981137


# work plans

1. figure out the largest dataset I can use for the project, what are they and where are they and how to use them?
2. figure out the MGMT and TMZ relation hold, do the test using previous method and collected public data.
3. work on the promoter activity predection software, and search for other available method and pipeline.
4. undertand the pedep project, general goal, the part charged by paul, and what could be my standing point.
5. figure out my specific project goal and working plan. by doing the literature research and reading, geting to do the newest progress in the erea and backgrounds. then writing a specific working plan in the form of grant.
6. working out a effectient project manage system, and improve the coding stills.
7. make progress every two weeks.

# Results summary  

Related paper note ID: [N202401251601_MethylationtoActivity](https://github.com/yz46606/paper_note/blob/main/N202401251601_MethylationtoActivity.md)

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4967469/pdf/main.pdf
https://www.cancerrxgene.org/downloads/bulk_download
https://www.cellosaurus.org/
https://depmap.sanger.ac.uk/documentation/datasets/methylation/



