`K202305171252` scan the arabidopsis genome for >1kb mCHG introns and identify how many of them are affected by methylation reduction
 
Key words:  `RNA-seq` `isoform quantification` `methylation intron` `long introns >1kb` `mutants` `ibm2` `cnt1`
 
# Question

Intron methylation and its effects on transcription is not specifically limited to IBM1, it would be interesting to scan the Arabidopsis genome
and find all mCHG methylated long introns (>1kb), and check how many of them have methylation reduction in cmt3/met1/suv456 mutants, and produce more
incomplete transcripts. how many of those genes also overlap with IBM2 mutants.

another exploring is to see how many of those methylation reduction introns also happened in cnt1 for example, and further more, behivious at population level.

# Analysis Steps

1.scan arabidopsis genome and find >1kb intron location  
2.check how many of them are enriched in mCHG  
3. how many of them shows reduction in mutant lines cmt3/met1/suv456/  
4. how many of those methylation reduction affect transcription?  
5. if they also show enriched incomplete transcription in ibm2 mutants?  
6. is there any common motif in the introns sequences regions that regulated by methylation.   
7. how many of those changes happenened in nature accessions?  

# Results summary  

Based on a analysis between suvh456 and wt, only "AT3G05410.2" "AT3G07610.3" are selected as positive markers that their expression affected by mCHG.
next, try to check the sequence similarity between AT3G05410 and AT3G07610!!!!!!!!!!!!!


AT1G58602
AT1G11270
which reported before in https://www.nature.com/articles/ncomms3301, were not detected in my analysis, probably because of definiation of long intron, not include them in the long intron gene list.
Related Work log ID: [W202305231510_scan1kb](https://github.com/yz46606/Working_record/blob/main/W202305231510_scan1kb.md)
